app/models/druid_version_zip.rb

```diff
@@ -82,7 +85,9 @@ def part_keys
   #  the last one will end .zip, so two parts is:  .z01, zip. (this agrees with zip utility)
   # @return [Array<Pathname>] Existing pathnames for zip parts based on glob (.zip, .z01, .z02, etc.)
   def part_paths
-    @part_paths ||= Pathname.glob(file_path.sub(/.zip\z/, '.z*'))
+    @part_paths ||= Pathname.glob(file_path.sub(/.zip\z/, '.z*')).reject do |path|
+      path.to_s =~ /.md5\z/
+    end
```

Comment:
This could also be a one-liner, with curly braces instead of the do/end, I think?

Reply:
rubocop complains for line length